tm2py/config.py

```diff
@@ -1152,6 +1151,35 @@ class TransitClassConfig(ConfigItem):
     required_mode_combo: Optional[Tuple[str, ...]] = Field(default=None)
 
 
+@dataclass(frozen=True)
+class ManualJourneyLevelsConfig(ConfigItem):
+     """Manual Journey Level Specification"""
+     level_id: int
+     group_fare_systems: Tuple[int, ...]
+
+
+@dataclass(frozen=True)
+class TransitJourneyLevelsConfig(ConfigItem):
+    """Transit manual journey levels structure."""
+    use_algorithm: bool = True
+    specify_manually: bool = False
+    manual: Optional[Tuple[ManualJourneyLevelsConfig, ...]] = None
```

Comment:
@vivverma9 
Please add a long docstring for this as well. Please include at least two examples in the docstring.